polybench/fdtd2d.cpp

```diff
@@ -198,6 +198,7 @@ class Polybench_Fdtd2d {
 
 int main(int argc, char** argv) {
 	BenchmarkApp app(argc, argv);
-	app.run<Polybench_Fdtd2d>();
-	return 0;
+        if(app.deviceSupportsFP64())
+          app.run<Polybench_Fdtd2d>();
+        return 0;

```

Comment:
Indentation looks odd.

Reply:
I just `clang-format`ed. Indentation and format is inconsistent all over sycl-bench. Not sure whether we should care too much.

Reply:
True, but in this case, seems unnecessary to indent the new snippet that much into `main`. Can we remove some of the spaces at the beginning of each line?

Reply:
Okey, I see the issue. That file's using tabs for some reason and I could not see any problem in my editor. Working nice now.